cli/src/cli_builder.rs

```diff
@@ -500,7 +504,8 @@ fn xxx<'a>(command: CliCommand) -> Option<&'a str> {
         | CliCommand::SiloMetric
         | CliCommand::SystemMetric
         | CliCommand::UserBuiltinList
-        | CliCommand::UserBuiltinView => None,
+        | CliCommand::UserBuiltinView
+        | CliCommand::PhysicalDiskView => None,
```

Comment:
can we add this? If we don't I'll never find it again. "system hardware disk view" and put it with its friends?

Reply:
Done in 22645eb:

```
bnaecker@shale : ~/oxide.rs $ ./target/release/oxide system hardware disk view
error: the following required arguments were not provided:
  --disk-id <disk-id>

Usage: oxide system hardware disk view --disk-id <disk-id>

For more information, try '--help'.
```

Reply:
Tests fixed in 671c3e1